azurelinuxagent/ga/persist_firewall_rules.py

```diff
@@ -71,23 +72,28 @@ class PersistFirewallRulesHandler(object):
     # modify the unit file on VM too
     _UNIT_VERSION = "1.4"
 
+    _DISTRO = get_distro()[0]
+
     @staticmethod
-    def get_service_file_path():
+    def get_service_file_path(mode=''):
         osutil = get_osutil()
         service_name = PersistFirewallRulesHandler._AGENT_NETWORK_SETUP_NAME_FORMAT.format(osutil.get_service_name())
-        return os.path.join(osutil.get_systemd_unit_file_install_path(), service_name)
+        if PersistFirewallRulesHandler._DISTRO == 'rhel' and mode != 'package_mode':
```

Comment:
I think if we define get_network_setup_service_install_path in the base class as returning get_systemd_unit_file_install_path, we would not need this change.

Reply:
Thanks! Change to use get_network_setup_service_install_path for all the distros.